src/index.html

```diff
@@ -217,6 +217,7 @@ <h1>Easy Steps To Finding Your Perfect Roommate</h1>
 		</template>
 		<template id="post-register">
 			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
+			<link rel="stylesheet" href="../src/styles/responsive.css">
```

Comment:
This should be just ```href="styles/responsive.css"```. The button is still visible on desktops screens which shouldn't be happening

Reply:
Make sure you test the button is not showing on laptop & desktop screen sizes which you can immitate in the browser developer console https://developers.google.com/web/tools/chrome-devtools/device-mode